src/services/barcode-readers/quagger.ts

```diff
@@ -17,10 +14,7 @@ const BARCODE_TYPES = [
   'code_93_reader',
 ]
 
-export default function (
-  imgData /* : string */,
-  barcodeTypes /* : string[] | void */,
-) /* : Promise<string | void> */ {
+export default function (imgData: string, barcodeTypes: string[] | void) {
```

Comment:
```suggestion
export default function (imgData: string, barcodeTypes?: string[]) {
```
Can we make this optional?
or else `undefined`? as `void` behaves differently to `undefined` in ts as oppposed to flow (no optional chaining on void, can't pass to other optional arg fns)
```suggestion
export default function (imgData: string, barcodeTypes: string[] | undefined) {
```